histoDB/histodb-dataserver/pom.xml

```diff
@@ -153,11 +153,16 @@
         </dependency>
 
         <dependency>
+            <groupId>eu.itesla_project</groupId>
+            <artifactId>ucte-import</artifactId>
+            <version>${ipstentsoe.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
I mean: without that dependency the build terminates without errors, but the dataserver "uber" jar  does not contain the importer class for the Ucte (note that also the cim1-import dependency is declared in this pom)